httpclient5/src/main/java/org/apache/hc/client5/http/auth/KerberosConfig.java

```diff
@@ -53,25 +49,28 @@ public enum Option {
 
     public static final KerberosConfig DEFAULT = new Builder().build();
 
-    private final Option stripPort;
-    private final Option useCanonicalHostname;
-    private final Option requestDelegCreds;
+    private final Option stripPort; //Effective default is ENABLE
+    private final Option useCanonicalHostname; //Effective default is ENABLE
+    private final Option requestDelegCreds; //Effective default is DISABLE
+    private final Option requestMutualAuth; //Effective default is DISABLE
```

Comment:
All I can came up with is:
- GSS does not default to mutual auth
- Backwards compatibility

I would at least leave an option to disable this, so that there is an out for very broken cases.
Of course there is also a case for forcing users to clean up their potentially broken setup.

Reply:
I have looked at this, and the current API is really built around using the gss library default settings.

Since we are making incompatible changes anyway, if we want to set non-default gss options by default, (and keep them configurable), then we should remove the default options, and use simple booleans with explicit true/false defaults.